components/ui/timeline.tsx

```diff
@@ -0,0 +1,42 @@
+type TimelineProps = {
+  active: string;
+};
+
+export function Timeline({ active }: TimelineProps) {
+  return (
+    <div className="flex flex-row items-center justify-center mt-2">
```

Comment:
How about making one circled word a functional component and using that three times for readability & maintainability?

Reply:
Looks great!